web/src/utils/icmp.ts

```diff
@@ -0,0 +1,75 @@
+/** TODO: remove this file, when replaced https://github.com/netobserv/flowlogs-pipeline/pull/429/files#r1219203830 */
```

Comment:
Similar to tcp drop approach, we could also apply this on FLP side

Reply:
Not sure what's the plan here; let's wait for feedback. 

We'll do a followup anyways if it's the case

Reply:
not sure to follow that discussion :-) what are the options?
is it about who is responsible to interpret the values: flp or plugin ?

Reply:
The TCP State & Cause [are translated to string in FLP](https://github.com/netobserv/flowlogs-pipeline/pull/429#discussion_r1219203830) whereas it's not the case for ICMP values.

I wonder if we should do it or not